src/modules/shared/components/Card/Card.tsx

```diff
@@ -0,0 +1,81 @@
+import clsx from 'clsx';
+import { FC, ReactNode } from 'react';
+
+import styles from './Card.module.scss';
+
+type Renderable = ReactNode;
+
+export interface CardProps {
+	edge?: 'zinc' | 'none';
+	image?: Renderable;
+	orientation?: 'horizontal' | 'vertical';
+	padding?: 'both' | 'content' | 'vertical' | 'none';
+	subtitle?: Renderable;
+	title?: Renderable;
+	toolbar?: Renderable;
+}
+
+const defaultProps: CardProps = {
+	edge: 'zinc',
+	orientation: 'vertical',
+};
+
+const Card: FC<CardProps> = ({
+	edge,
+	children,
+	image,
+	orientation,
+	padding,
+	subtitle,
+	title,
+	toolbar,
+}) => {
+	return (
+		<article
+			className={clsx(
+				styles['c-card'],
+				styles[`c-card--edge-${edge}`],
+				styles[`c-card--orientation-${orientation}`],
+				styles[`c-card--padded-${padding}`]
+			)}
+		>
+			<section className={clsx(styles['c-card__top-wrapper'])}>
+				{image && (
+					<section className={clsx(styles['c-card__image-wrapper'])}>
+						{typeof image === 'string' ? (
+							<img className='u-image-responsive' src={image} alt={title?.toString() || "The card's image"} /> //eslint-disable-line
+						) : (
+							image
+						)}
+					</section>
+				)}
+			</section>
+
+			<section className={clsx(styles['c-card__bottom-wrapper'])}>
+				<section className={clsx(styles['c-card__header-wrapper'])}>
+					{title && <div className={clsx(styles['c-card__title-wrapper'])}>{title}</div>}
+
+					{toolbar && (
+						<div className={clsx(styles['c-card__toolbar-wrapper'])}>{toolbar}</div>
+					)}
+				</section>
+
+				{subtitle && (
+					<section className={clsx(styles['c-card__subtitle-wrapper'])}>
+						{subtitle}
+					</section>
+				)}
+
+				{children && (
+					<section className={clsx(styles['c-card__children-wrapper'])}>
+						{children}
+					</section>
+				)}
+			</section>
+		</article>
```

Comment:
Ik zou van de wrapping elements telkens een `div` maken omdat die semantisch irrelevant zijn. Een `Section` verwacht een h-tag en gaat een nieuw stuk content aankondigen dat in principe op zichzelf kan staan. 
Titel zou idealiter een h-tag moeten zijn, maar ik weet niet als die altijd hetzelfde zal zijn. In dat geval zou ik er een p-tag van maken. 
Ik zou ook de ondertitel in een p-tag zetten.

Reply:
Second-level `section`-elementen omgevormd naar `div`'s.

Titel & subtitel moeten generiek blijven, het is aan implementerende componenten om de juist `<hx>`, `<b>` of `<span>` tag te implementeren.